### PR TITLE
(docs) : importing additional info from docs.openebs.io (2.11.0)

### PR DESCRIPTION
**Pull request by rakeshPRaghu**
This PR will import the Additional info section (2.11.0) from docs.openebs.io
Signed-off-by: Rakesh PR <rakesh.pr@mayadata.io>

### CODE REVIEW

File: docs/src/theme/DocPage/styles.module.scss [hunk 1]
```diff
@@ -67,11 +67,11 @@
   }
 
   .docPage {
-    grid-template-columns: var(--doc-sidebar-width) 1fr;
+    grid-template-columns: var(--doc-sidebar-width) minmax(0, 1fr);
   }
 
   .colMainContent {
-    width: calc(100vw - var(--doc-sidebar-width));
```
Comment (IsAmrish) [inline docs/src/theme/DocPage/styles.module.scss:74]: why change in style?

File: docs/versioned_docs/version-2.11.0/additional-info/faq.md [hunk 2]
```diff
@@ -0,0 +1,662 @@
...
+
+[Does OpenEBS support encryption at rest?](#encryption-rest)
+
+[Can the same BDC name be used for claiming a new block device?](#same-bdc-claim-new-bd)
+
+-----
+
+<font size="6" color="blue">General</font>
+
+### What is most distinctive about the OpenEBS architecture?
```
Comment (IsAmrish) [inline docs/versioned_docs/version-2.11.0/additional-info/faq.md:86]:
```suggestion
### What is most distinctive about the OpenEBS architecture? {#What-is-most-distinctive-about-the-OpenEBS-architecture?}
```

Comment (IsAmrish) [reply docs/versioned_docs/version-2.11.0/additional-info/faq.md:86]: if you look carefully, id is not fully `lowercased`

File: docs/versioned_docs/version-2.11.0/additional-info/faq.md [hunk 3]
```diff
@@ -0,0 +1,662 @@
...
+
+[Go to top](#top)
+
+### How does OpenEBS provide high availability for stateful workloads?
+
+An OpenEBS Jiva volume is a controller deployed during OpenEBS installation. Volume replicas are defined by the parameter that you set. The controller is an iSCSI target while the replicas play the role of a disk. The controller exposes the iSCSI target while the actual data is written. The controller and each replica run inside a dedicated container. An OpenEBS Jiva volume controller exists as a single instance, but there can be multiple instances of OpenEBS Jiva volume replicas. Persistent data is synchronized between replicas. OpenEBS Jiva volume high availability is based on various scenarios as explained in the following sections. 
+
+**Note:** Each replica is scheduled in a unique K8s node, and a K8s node never has two replicas of one OpenEBS volume.
+
+* **What happens when an OpenEBS volume controller pod crashes?**
```
Comment (IsAmrish) [inline docs/versioned_docs/version-2.11.0/additional-info/faq.md:356]: why this extra `*`?
is it a list ?

Comment (rakeshPRaghu) [reply docs/versioned_docs/version-2.11.0/additional-info/faq.md:356]: yes

**Review by IsAmrish — APPROVED** [review]

**Review by satyapriyamishra222 — APPROVED** [review]

**Pull request merged**